pkg/analytics/analytics.go

```diff
@@ -0,0 +1,105 @@
+package analytics
+
+import (
+	"encoding/json"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// Variables that we'll set @ build time
+var (
+	Tier    = "oss"
+	Version = "v0.0.0"
```

Comment:
We have too many Version constants already, let's not add even more - `cmd/gitops/version.Version` is perfectly adequate for this.

Reply:
Imported it instead of setting the value in analytics.